src/sentry/api/endpoints/organization_events_spans_performance.py

```diff
@@ -666,6 +732,33 @@ def get_example_transaction(
         if span["op"] == span_op and int(span["hash"], 16) == span_group_id
     ]
 
+    if min_exclusive_time is not None:
```

Comment:
something analogous to the above should be easier to read. iterating the list of spans up to 3 times isn't that bad in terms of performance and keeps the code much cleaner

Reply:
done!